### ISSUE TITLE
CI automation to keep models site up to date

### ISSUE THREAD

**Issue by stevelr** [wasmCloud/weld#3]
The wasmcloud/models repo contains html files and downloadable smithy model files. This repo (weld, and/or wherever shared smithy files end up living) needs to have CI tools to copy changed smithy files into the correct locations in the models repo.

Depends on Issue #2

**Comment by stevelr** [comment]
also update index.html using a handlebars template.

**Comment by stevelr** [comment]
Bill - assigning this to you - we'll need to talk about it so I can describe the issue

**Comment by stevelr** [comment]
duplicate of interfaces #2
